src/server/game/Entities/Player/Player.cpp

```diff
@@ -23968,8 +23968,11 @@ void Player::SetViewpoint(WorldObject* target, bool apply)
         // farsight dynobj or puppet may be very far away
         UpdateVisibilityOf(target);
 
-        if (target->isType(TYPEMASK_UNIT) && target != GetVehicleBase())
+        if (target->isType(TYPEMASK_SEER) && target != GetVehicleBase())
```

Comment:
Other than my change and its definition, `TYPEMASK_SEER` is only used at one point in `Player::GetViewpoint`. Is there any situation that would set the player's view to a dynamic object such that `Player::GetViewpoint` would return a dynamic object? If setting a viewpoint to a dynamic object would be a server-crasher. If it is not the case that `Player::GetViewpoint` would ever return a dynamic object, I'd like to remove `TYPEMASK_DYNAMICOBJECT` from `TYPEMASK_SEER`.